langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaToolChatModelIT.java

```diff
@@ -121,6 +137,58 @@ void should_handle_tool_without_parameter() {
 
     }
 
+    @Test
+    void should_handle_tools_call_in_streaming_scenario() throws Exception {
```

Comment:
@bidek , wpuld you mind to add also a unit test, please? 
It is much easier to handle corner case scenarios with unit tests